src/listeners/command-handler/CoreMessageParser.ts

```diff
@@ -16,16 +16,16 @@ export class CoreListener extends Listener<typeof Events.PreMessageParsed> {
 		if (!canRun) return;
 
 		let prefix = null;
-		const mentionPrefix = this.getMentionPrefix(message);
+		const disableMentionPrefix = this.getMentionPrefix(message);
 		const { client } = this.container;
 		const { regexPrefix } = client.options;
-		if (mentionPrefix) {
-			if (message.content.length === mentionPrefix.length) {
+		if (disableMentionPrefix) {
+			if (message.content.length === disableMentionPrefix.length) {
 				client.emit(Events.MentionPrefixOnly, message);
 				return;
 			}
 
-			prefix = mentionPrefix;
+			prefix = disableMentionPrefix;
```

Comment:
Revert all of these. These renames are invalid. It's still the `mentionPrefix`, except that it can also be `null`.  Below are the reverts as suggestions so you can apply them directly in GH.

Reply:
Oh my bad, i used the vscode search and replace, must have been it capturing those too 😅